mockgcp/fixup-third-party.sh

```diff
@@ -25,13 +25,17 @@ rm -rf mockgcp/ && mkdir -p mockgcp/api/
 
 mv google/cloud/ mockgcp/
 mv google/iam/ mockgcp/
+mv google/api/apikeys/ mockgcp/api/
 mv google/api/serviceusage/ mockgcp/api/
 
 cd mockgcp
 
 find . -type f -print0 | xargs -0 sed -i -e "s@google/iam/@mockgcp/iam/@g"
 find . -type f -print0 | xargs -0 sed -i -e "s@google\.iam@mockgcp.iam@g"
 
+find . -type f -print0 | xargs -0 sed -i -e "s@google/api/apikeys/@mockgcp/api/apikeys/@g"
```

Comment:
Just for self-education: Is it a required step when adding a mock API? I'm not sure if I've missed anything, but I didn't find them in the [README](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/mockgcp/README.md).

Reply:
This is just when we add a new API outside of the normal google/cloud group.

However, I'll update the README!